package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@editorjs/footnotes",
+  "version": "1.0.0",
+  "description": "Footnotes Block Tune for Editor.js",
+  "main": "dist/bundle.js",
+  "repository": "https://github.com/editor-js/footnotes",
+  "author": "CodeX Team <team@codex.so>",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development --watch",
+    "lint": "eslint src/ --ext .ts"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.16",
+    "@babel/preset-env": "^7.13.15",
+    "@editorjs/editorjs": "https://github.com/codex-team/editor.js#feature/tunes-improvements",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^5.2.4",
+    "eslint": "^7.24.0",
+    "eslint-config-codex": "^1.6.1",
+    "eslint-loader": "^4.0.2",
+    "postcss-loader": "^5.2.0",
+    "postcss-nested": "^5.0.5",
+    "postcss-nested-ancestors": "^2.0.0",
+    "raw-loader": "^4.0.2",
+    "style-loader": "^2.0.0",
+    "ts-loader": "^9.1.0",
+    "typescript": "^4.2.4",
+    "webpack": "^5.35.0",
+    "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "@codexteam/shortcuts": "^1.1.1"
```

Comment:
need to move this to devdeps too, as i understand. because you are going to use bundle not sources

Reply:
It is included to bundle, so should be in deps